CppImport/src/Comment.cpp

```diff
@@ -78,8 +78,41 @@ void Comment::removeFromItemList()
 QString Comment::text()
 {
 	auto lines = text_.split('\n');
-	for (auto i = 0; i < lines.length(); i++)
-		lines[i] = lines[i].trimmed();
+	if (!lines.first().startsWith("/*"))
+	{
+		QRegularExpression singleLineRegex{"^\\s*//\\s*([^\\s].*)$", QRegularExpression::DotMatchesEverythingOption};
```

Comment:
why the second `\s*` ? Shouldn't that be just `\s?` And why not simply `(.*)$` afterwards?

And do you need the `DotMatchesEverythingOption`? Isn't that just in the case where you want the `.` to match new lines? Do we care about that?